.github/workflows/webrtc_videoroom_advanced_sandbox_build_and_deploy.yml

```diff
@@ -53,8 +53,8 @@ jobs:
       
       - name: Change Docker-Compose address before deployment
         id: sed_it_again
-        run: sed -i -e "s/demo-/sandbox-/g" webrtc/videoroom_advanced/docker-compose.yml
-
+        run: sed -i -e "6,9 s/videoroom/sandbox-videoroom/" webrtc/videoroom_advanced/docker-compose.yml
+        
```

Comment:
What's that?

Reply:
This needed to use this same values of `VIRTUAL_HOST` and `LETSENCRYPT_HOST` in `docker-compose.yml` on demo and on sandbox